routing/joint_segment.cpp

```diff
@@ -6,7 +6,7 @@
 
 namespace routing
 {
-
+bool IsRealSegment(Segment const & segment) { return segment.GetSegmentIdx() != std::numeric_limits<uint32_t>::max(); }
```

Comment:
anonymous namespace?
clang format pls. (too long line)

Reply:
useless, deleted it